impeller/playground/playground.cc

```diff
@@ -139,6 +139,9 @@ void Playground::SetupWindow() {
 }
 
 void Playground::TeardownWindow() {
+  if (context_) {
```

Comment:
This is my hacky take on making sure the concurrent message loop is torn down on the raster thread and not a worker thread